### PR TITLE
Add user stories

### PR DESCRIPTION
**Pull request by gohkh**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -286,9 +286,50 @@ Refer to the guide <<DevOps#, here>>.
...
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
+|Priority |As a ... |I want to ... |So that ...
+|`* * *` |beginner cook |find new recipes easily |I don't waste time searching though recipes from different sources
+
+|`* * *` |regular cook |record my own recipes |I can refer to them easily in future
+
+|`* * *` |forgetful person |add ingredients for my planned meals to a grocery list easily |I know what I need to get when shopping
+
+|`* * *` |disorganized person |keep track of the ingredients I have at home |plan my meals better
```
Comment (teo-jun-xiong) [inline docs/DeveloperGuide.adoc]: ... at home| I can plan my meals better

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -286,9 +286,50 @@ Refer to the guide <<DevOps#, here>>.
...
+
+|`* * *` |low-income individual |cook a meal with the ingredients I already have |I can save money
+
+|`* * *` |person with seafood allergy |cook meals that don't contain seafood |I do not have an allergic reaction
+
+|`* *` |cooking enthusiast |edit recipes |I can tweak a recipe to my liking
+
+|`* *` |regular cook |set a timer during meal preparation |I can control the quality of my meal
+
+|`* *` |CS student |cook a quick meal |spend more doing CS2103T
```
Comment (teo-jun-xiong) [inline docs/DeveloperGuide.adoc]: ... quick meal |I can spend more time doing CS2103T

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -286,9 +286,50 @@ Refer to the guide <<DevOps#, here>>.
...
+
+|`*` |health-conscious person |keep track of the nutritional value of the food I eat |I can meet my nutritional goals
+
+|`*` |regular gym-goer |keep track of my dietary intake |I can meet my fitness goals
+
+|`*` |obesity fighter |keep track of my calorie and fat intake |I can lose weight
+
+|`*` |stay-at-home parent |plan a variety of meals for the week |I can make sure that my family eats healthily
+
+|`*` |kiasu parent |know how much ingredients I need for 2 weeks |my family can survive an epidemic
```
Comment (teo-jun-xiong) [inline docs/DeveloperGuide.adoc:328]: ... 2 weeks |My family can survive an epidemic

Comment (gohkh) [reply docs/DeveloperGuide.adoc:328]: I'm starting with small letters, just that the rest start with "I"

**Review by teo-jun-xiong — APPROVED** [review]
LGTM

**Pull request merged**